### PR TITLE
Upgrade SQLX from 0.7 to 0.8

### LINKED ISSUE #223: Upgrade SQLX from 0.7 to 0.8
There is a new version of SQLX.

### PR DESCRIPTION
**Pull request by sergiomeneses**
closes #223

### CODE REVIEW

File: Cargo.toml
Comment (nyurik) [inline Cargo.toml:33]: @sergiomeneses please disable or adjust your toml auto-formatter - while i usually like those, the newlines it introduces here and above are not very common in rust -- especially in the dependencies list.

Comment (sergiomeneses) [reply Cargo.toml]: Done.

File: geozero/src/postgis/postgis_sqlx.rs
Comment (nyurik) [inline geozero/src/postgis/postgis_sqlx.rs:147]: so my understanding here is that sqlx 0.8 modified their Encode trait to handle boxed errors? If so, seems reasonable

Comment (sergiomeneses) [reply geozero/src/postgis/postgis_sqlx.rs]: I update this to qualified use

**Comment by sergiomeneses** [comment]
fix workflow errors.

**Comment by sergiomeneses** [comment]
I think is ready for review.

**Review by nyurik — APPROVED** [review]
@sergiomeneses looks good, thanks! I will merge it now with a minor separate fix.  For the future - when you create a PR, make sure to create a branch first.  This way maintainers can make changes in that branch before merging.

**Pull request merged**
